traitcuration/settings.py

```diff
@@ -44,11 +44,18 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'django.contrib.sites',
     'compressor',
     'django_celery_results',
-    'celery_progress'
+    'celery_progress',
+    'allauth',
+    'allauth.account',
+    'allauth.socialaccount',
+    'allauth.socialaccount.providers.google',
 ]
 
+SITE_ID = 2
```

Comment:
What does this do, and why is this “2”?

Reply:
So in order for the auth package to work, I have to register the domain in the app's database, you can see it in the `Site` model in the Django Admin. It already has an example site there which I left as is during development and added another entry with the actual domain, hence the id of 2. I forgot I still had that as 2 so I will change it back to 1 and modify the database in the deployment accordingly

Reply:
If it's not too difficult, let's change it to 1 and get rid of the example site entities. Also, could you please add your explanation (or maybe just the gist of it) as a comment for this `SITE_ID` assignment?